jobs/apptypes_test.groovy

```diff
@@ -0,0 +1,63 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+
+name = "apptypes_e2e"
+
+
+  job(name) {
+    description """
+      runs a e2e-runner job nightly we have to setup TEST=bps env variable for the docker container to execute the test
```

Comment:
Can we improve this job description so that the purpose is more readily understood?  Meaning, instead of going into the particulars of `setup TEST=bps env variable for the docker container…` how about describing it more generally, something like `Runs the buildpack and dockerfile sub-suites of e2e tests on a nightly basis`